spec/requests/quizzes_spec.rb

```diff
@@ -0,0 +1,97 @@
+require "rails_helper"
+
+RSpec.describe "Quizzes", type: :request do
+  describe "GET index" do
+    before do
+      Quiz.create(title: "First Quiz", description: "This is the first quiz")
+      Quiz.create(title: "Second Quiz", description: "This is the second quiz")
+      Quiz.create(title: "Third Quiz", description: "This is the third quiz")
+      Quiz.create(title: "Fourth Quiz", description: "This is the fourth quiz")
+    end
+
+    it "gets all quizzes and checks the content" do
+        get "/quizzes"
+        expect(response).to have_http_status(:success)
+        expect(response.body).to include("First Quiz")
+        expect(response.body).to include("Second Quiz")
+        expect(response.body).to include("Third Quiz")
+        expect(response.body).to include("Fourth Quiz")
+      end
+  end
+
+  describe "GET show" do
+    before do
+        Quiz.create(title: "First Quiz", description: "This is the first quiz")
```

Comment:
```suggestion
        @quiz = Quiz.create(title: "First Quiz", description: "This is the first quiz")
```

We should set this as a variable so we can get the id from it rather than relying on it creating a record with an id of 1 (as that depends on database state)

Reply:
That makes sense - thanks for the change!